k8s/upgrades/0.8.0-0.8.1/cstor_pool_upgrade.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+set -e
+
+##################################################################
+# STEP: Get SPC name as argument                                 #
+#                                                                #
+# NOTES: Obtain the pool deployments to perform upgrade operation#
+##################################################################
+
+function usage() {
+		echo
+		echo "Usage:"
+		echo
+		echo "$0 <spc-name> <openebs-namespace>"
+		echo
+		echo "  <spc-name> Get the SPC name using: kubectl get spc"
+		echo "  <openebs-namespace> Get the namespace where pool pods"
+		echo "    corresponding to SPC are deployed"
+		exit 1
+}
+
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+spc=$1
+ns=$2
+
+##Checking the version of OpenEBS ####
+openebs_version=$(kubectl get csp -o jsonpath="{.items[0].metadata.labels.openebs\.io/version}")
+if [ $openebs_version != "0.8.0" ]; then
+		echo "Current replica version is not 0.8.0";exit 1;
+fi
+
+###Get the no.of pool pods are running #######
+pool_cnt=$(kubectl get po -n $ns -l app=cstor-pool,openebs.io/storage-pool-claim=$spc -o jsonpath='{.items[?(@.status.phase=="Running")].metadata.name}'| wc -w)
+
+# Get the list of pool deployments for given SPC, delimited by ':'
+pool_deploys=`kubectl get deploy -n $ns \
+ -l app=cstor-pool,openebs.io/storage-pool-claim=$spc \
+ -o jsonpath="{range .items[*]}{@.metadata.name}:{end}"`
+
+echo "Patching the csp resource"
+for csp_res in `echo $pool_deploys | tr ":" " "`; do
```

Comment:
don't assume pool_deploy is same as csp resource.. try to get the csp resource name

Reply:
It is same @vishnuitta .
If it is not same then it is a bug.

Reply:
But as said by @vishnuitta let us fetch those names using labels